test/test.js

```diff
@@ -1,13 +1,26 @@
+"use strict";
 import test from 'ava';
 
 import 'babel-register';
 import 'babel-polyfill';
 
 import AnkiExport from '../src/index';
 import fs from 'fs';
+import  decompress from 'decompress';
+import _ from 'lodash';
```

Comment:
Please change to `import sortBy from 'lodash.sortby';`. See below why `isEqual` is not needed